flink-runtime/src/test/java/org/apache/flink/runtime/util/StartupUtils.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.runtime.util;
+
+import java.util.List;
+
+public class StartupUtils {
+	
+	/**
+	 * A utility method to analyze the exceptions and collect the clauses
+	 * 
+	 * @param e  the root exception (Throwable) object
+	 * @param causes  the list of exceptions that caused the root exceptions
+	 * @return  a list of Throwable
+	 */
+	public List<Throwable> getExceptionCauses(Throwable e, List<Throwable> causes) {
```

Comment:
I think it would be slightly better if we made this method `static`. Then we wouldn't have to create the `StartupUtils` objects in the different test classes.